RNTester/RNTesterPods.xcodeproj/project.pbxproj

```diff
@@ -738,7 +738,7 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_CXX_LANGUAGE_STANDARD = "c++14";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = KMCB293693;
```

Comment:
The development team is specific to your Apple ID account, and shouldn't be committed. I know this is super annoying as Xcode does it automatically for you.